README.md

```diff
@@ -2,3 +2,7 @@
 Test repository
 
 My first git commit
+
+new line 1
+
```

Comment:
please remove extra line here

Reply:
these extra lines added for more clarity

Reply:
okey then please meake the first letter caps

Reply:
ok I'll do it now